app/layout.tsx

```diff
@@ -0,0 +1,28 @@
+import { apiPlugin, storyblokInit } from '@storyblok/react/rsc'
+import { ReactElement } from 'react'
+import StoryblokProvider from '../components/StoryblokProvider'
+import '../styles/globals.scss'
+
+storyblokInit({
```

Comment:
bruchts das hie, wes när im provider ono mau gmacht wird?
i gseh das dr Provider eh Client-Komponente isch, aber gliich nimmts mi wunger obs das hie brucht 😅

Reply:
Hani mi ou gfrägt :) Wennis wäglah flügts of d Frässi ond hie heisis ou 2x dinne: https://github.com/storyblok/storyblok-react/tree/main/playground-next13-live-editing